src/Message/Communicate.cpp

```diff
@@ -15,14 +15,14 @@
 //////////////////////////////////////////////////////////////////////////////////////
 
 
-#include <Configuration.h>
 #include "Message/Communicate.h"
-#include "Message/TagMaker.h"
 #include <iostream>
 #include <cstdio>
-#include <Platforms/sysutil.h>
-#include <Utilities/FairDivide.h>
 #include <fstream>
+#include "config.h"
+#include "Message/TagMaker.h"
```

Comment:
It looks like the TagMaker class is unused (?)  Maybe we could remove it?

Reply:
Deleted.